### DISCUSSION TITLE
Apply tags to existing image in order to promote

### DISCUSSION THREAD

**Discussion by Wazbat** [docker/metadata-action#331] [Q&A]
Hi there! This action works great, and we use it extensively with `docker/build-push-action`, however I had a question on how best to do this workflow

When creating a new tag, I want to apply the generated metadata to the already existing image that was created on last push, in order to **promote** that image. The image will be accessible using the `sha` of the tag thanks to this action, however how can I effectively apply the output of this action, tags and metadata, to the image I just pulled, that was generated previously?

**Comment by Wazbat** [comment]
Has anyone manage this? I haven't had a chance to revisit it but I'm curious if anyone else is following this strategy

**Comment by marienpflaum** [comment]
It would be nice if this command had a github action:

https://docs.docker.com/reference/cli/docker/image/tag/#tag-an-image-for-a-private-registry

That would solve it i guess. I am searching something like this to,

For now I created an upgrade workflow which just repushes

**Reply by crazy-max** [reply]
If anything we would use `buildx imagetools`: https://github.com/docker/actions-toolkit/issues/239 but that's offtopic with this action.
